dataland-keycloak/realms/datalandsecurity-realm.json

```diff
@@ -700,7 +700,7 @@
       "TERMS_AND_CONDITIONS"
     ],
     "realmRoles": [
-      "ROLE_USER",
+      "ROLE_ADMIN",
```

Comment:
why is this role change necessary?

Reply:
Das sind die Rollen des backend-service-accounts bei Keycloak.

Im `main`-code kann das backend ja selbst prüfen, ob jemand ein company-owner ist.

Nach meinem change jetzt, muss das backend aber den community-manager fragen, ob jemand company-owner ist.
Das kann man aber nur als keycloak-admin.  Deswegen musste ich dem backend-service-account jetzt admin-Rechte in keycloak geben.